src/command/info/info.c

```diff
@@ -708,6 +724,16 @@ formatTextDb(const KeyValue *stanzaInfo, String *resultStr, const String *backup
 
                     strCat(backupResult, LF_STR);
                 }
+
+                if (kvGet(backupInfo, BACKUP_KEY_CHECKSUM_PAGE_ERROR_LIST_VAR) != NULL)
+                {
+                    StringList *checksumPageErrorList = strLstNewVarLst(
+                        varVarLst(kvGet(backupInfo, BACKUP_KEY_CHECKSUM_PAGE_ERROR_LIST_VAR)));
+
+                    strCatFmt(
+                        backupResult, "            files with page checksum error: %s\n",
```

Comment:
I would removed `files with` - we like to keep the output short and `page checksum error` is more than sufficient to convey what is being displayed.

Reply:
Fixed, thanks.